packages/meditrak-server/src/tests/routes/exportSurveyResponses/exportSurveyResponses.test.js

```diff
@@ -0,0 +1,178 @@
+  const headerRow = aoaExportData[0];
+
+  for (let i = 0; i < INFO_COLUMN_HEADERS.length; i++) {
+    const headerCell = headerRow[i];
+    expect(headerCell).to.be.equal(INFO_COLUMN_HEADERS[i]);
+  }
+};
+
+describe('exportSurveys(): GET export/surveysResponses', () => {
+  const app = new TestableApp();
+  const models = app.models;
+
+  describe('Test permissions when exporting survey responses', async () => {
+    let vanuatuCountry;
+    let survey1;
+    let survey2;
+
+    before(async () => {
+      sinon.stub(xlsx.utils, 'aoa_to_sheet');
+
+      const adminPermissionGroup = await models.permissionGroup.findOne({ name: 'Admin' });
+      const donorPermissionGroup = await models.permissionGroup.findOne({ name: 'Donor' });
+
+      vanuatuCountry = await models.country.findOne({ code: 'VU' });
+
+      [{ survey: survey1 }, { survey: survey2 }] = await buildAndInsertSurveys(models, [
+        {
+          code: TEST_SURVEY_1_CODE,
+          name: TEST_SURVEY_1_NAME,
+          country_ids: [vanuatuCountry.id],
+          permission_group_id: adminPermissionGroup.id,
+        },
+        {
+          code: TEST_SURVEY_2_CODE,
+          name: TEST_SURVEY_2_NAME,
+          country_ids: [vanuatuCountry.id],
+          permission_group_id: donorPermissionGroup.id,
+        },
+      ]);
+
+      await buildAndInsertSurveyResponses(models, [
+        {
+          surveyCode: survey1.code,
+          entityCode: 'VU',
+          submission_time: new Date(),
+          answers: {
+            question_1_test: 'question_1_test answer',
+            question_2_test: 'question_2_test answer',
+          },
+        },
+        {
+          surveyCode: survey1.code,
+          entityCode: 'VU',
+          submission_time: new Date(),
+          answers: {
+            question_3_test: 'question_3_test answer',
+            question_4_test: 'question_4_test answer',
+          },
+        },
+        {
+          surveyCode: survey2.code,
+          entityCode: 'VU',
+          submission_time: new Date(),
+          answers: {
+            question_5_test: 'question_5_test answer',
+            question_6_test: 'question_6_test answer',
+          },
+        },
+        {
+          surveyCode: survey2.code,
+          entityCode: 'KI',
+          submission_time: new Date(),
+          answers: {
+            question_7_test: 'question_7_test answer',
+            question_8_test: 'question_8_test answer',
+          },
+        },
+      ]);
+    });
+
+    after(() => {
+      xlsx.utils.aoa_to_sheet.restore();
+    });
+
+    afterEach(() => {
+      Authenticator.prototype.getAccessPolicyForUser.restore();
+      xlsx.utils.aoa_to_sheet.resetHistory();
+    });
+
+    it('Sufficient permissions: Should allow exporting an existing survey if users have both Tupaia Admin Panel and survey permission group access to the country of that survey', async () => {
+      await prepareStubAndAuthenticate(app);
```

Comment:
To make the commonalities and differences more obvious between this and the following test case, I suggest using a structure like the following:

```js
    describe('Should allow exporting a survey if users have Tupaia Admin Panel and survey permission group access to the corresponding countries', () => {
      it('one country', () => {
        /* ... */
      });

      it('multiple countries', () => {
        /* ... */
      });
    });
```